interface/forms/procedure_order/new.php

```diff
@@ -536,110 +536,120 @@ function validate(f) {
 $oemr_ui = new OemrUI($arrOeUiSettings);
 ?>
 </head>
-<body class="body_top">
-<div id="container_div" class="<?php echo attr($oemr_ui->oeContainer()); ?>">
+<body>
+<div id="container_div" class="<?php echo attr($oemr_ui->oeContainer()); ?> mt-3">
     <div class="row">
         <div class="col-sm-12">
-            <div class="clearfix">
-                <?php echo  $oemr_ui->pageHeading() . "\r\n"; ?>
-            </div>
+            <?php echo  $oemr_ui->pageHeading() . "\r\n"; ?>
         </div>
     </div>
     <div class="row">
         <div class="col-sm-12">
             <form class="form-horizontal" method="post" action="<?php echo $rootdir ?>/forms/procedure_order/new.php?id=<?php echo attr_url($formid); ?>" onsubmit="return validate(this)">
                 <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
-                <fieldset>
+                <div class="container">
```

Comment:
This doesn't actually work. You have to have the fieldset tag to allow the legend tag to work properly. So the first tag that should be here is the fieldset tag followed by the div container.